local-volume/provisioner/pkg/util/volume_util.go

```diff
@@ -106,14 +106,17 @@ func (u *volumeUtil) DeleteContents(fullPath string) error {
 
 // GetFsAvailableByte returns available capacity in byte about a mounted filesystem.
 // fullPath is the pathname of any file within the mounted filesystem. Capacity
-// returned here is total capacity available to non-root users, and does not include
-// fs reserved space.
+// returned here is total capacity available.
 func (u *volumeUtil) GetFsAvailableByte(fullPath string) (uint64, error) {
-	var s unix.Statfs_t
-	if err := unix.Statfs(fullPath, &s); err != nil {
+	//largely copied from kubernetes/pkg/volume/util/fs.go:FsInfo(path)
```

Comment:
we most likely can't (yet) because it imports "k8s.io/kubernetes/pkg/api."

but yes --strip-vendor/-v is the correct flag needed to remove nested dependencies & to flatten everything